svg-aam/name/comp_labelledby.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html>
+<head>
+  <title>Name Comp: Labelledby</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<h1>SVG-AAM: Labelledby Tests</h1>
+
+<p>Tests SVG-specific aria-labelledby rules in <a href="https://w3c.github.io/svg-aam/#mapping_additional_nd">SVG-AAM §8.1 Name and Description</a>, but note the open issues in <a href="https://github.com/w3c/svg-aam/issues/31">SVG-AAM #31</a>.
+
+<span id="huey">Huey</span>
+<span id="dewey">Dewey</span>
+<span id="louie">Louie</span>
+<span id="scrooge">Scrooge</span>
```

Comment:
What does it say about Adam that aria-label = heroic swordsmen, but aria-labelledby = cartoon ducks?

Reply:
> What does it say about Adam that aria-label = heroic swordsmen, but aria-labelledby = cartoon ducks?

That I have good taste in entertainment. 😏